stable_baselines3/__init__.py

```diff
@@ -4,6 +4,7 @@
 from stable_baselines3.ppo import PPO
 from stable_baselines3.sac import SAC
 from stable_baselines3.td3 import TD3
+from stable_baselines3.her import create_her
```

Comment:
My thinking was, if it just inherits from OffPolicyAlgorithm like in SB2, the model_class (SAC, etc.) will have to be a separate attribute of the HER object, say `self.model = SAC()` and there's a lot of unnecessary duplication since both `self` and `self.model` inherit from `OffPolicyAlgorithm`. The dynamic inheritance here seems ugly but I thought once the HER class is created, it behaves like any other algo and the inheritance makes it cleaner to use the model_class's methods. Also, HER feels more like a wrapper which can be used with any off-policy algo  than a separate algo so I thought this wasn't too bad.